addons/sale_project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -5,12 +5,16 @@
         <xpath expr="//ProjectRightSidePanelSection[@name=&quot;'profitability'&quot;]" position="before">
             <ProjectRightSidePanelSection
                 name="'sales'"
-                show="state.data.sale_items and state.data.sale_items.total &gt; 0"
+                show="!!state.data.sale_items"
+                showData = "state.data.sale_items and state.data.sale_items.total &gt; 0"
             >
                 <t t-set-slot="title" owl="1">
                     Sales
                 </t>
-                <table class="table table-striped table-hover mb-4">
+                <t t-set-slot="helper" owl="1">
+                    Track what has been sold, delivered, and invoiced.
+                </t>
+                <table class="table table-striped table-hover mb-4" t-if="state.data.sale_items and state.data.sale_items.total &gt; 0">
```

Comment:
- Why not use `showData` i/o copying the condition?
- Why doesn't that behave the same way as in `addons/project/static/src/components/project_right_side_panel/components/project_right_side_panel_section.xml`? (that is : if helper and not showData => helpder, else the table)